src/main/java/pl/net/rogala/eventy/repository/EventRepository.java

```diff
@@ -2,13 +2,24 @@
 
 import org.springframework.data.domain.Sort;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 import pl.net.rogala.eventy.entity.Event;
 
+import java.time.LocalDate;
+import java.util.List;
 import java.util.Set;
 
 @Repository
 public interface EventRepository extends JpaRepository<Event, Long> {
 
+//    @Modifying
```

Comment:
Jeżeli nie używane, wywalić, po to mamy gita, żeby nie chomikować kodu w komentarzach :-)

Reply:
Wiem, nie posprzątałam :)